.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "dropwizard-swagger"]
```

Comment:
I cannot remember the method by which I added this, but I have removed the submodule from .gitmodules - I believe this should suffice.